### PR TITLE
fix(telephony.alias): undo deletion of used folder

### PR DESCRIPTION
**Pull request by JeremyDec**
# Undo deletion of used component folder for ovhPabx members
MANAGER-2031

## Description of the change
Undo deletion folder by error.

For more details, see this PR (and this particular commit : https://github.com/ovh-ux/ovh-manager-telecom/pull/1098/commits/35c149b9708faf850ab61bfd75130a278807c696)

No refactor is done, and will be made once the design/feature will be revisited

### CODE REVIEW

File: client/components/telecom/telephony/alias/members/telecom-telephony-alias-members-add-modal.component.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+angular.module('managerApp').controller('telecomTelephonyAliasMembersAddModal', class telecomTelephonyAliasMembersAddModal {
```
Comment (JeremyDec) [inline client/components/telecom/telephony/alias/members/telecom-telephony-alias-members-add-modal.component.js]: No refactor is done, and will be made once the design/feature will be revisited

Comment (FredericEspiau) [reply client/components/telecom/telephony/alias/members/telecom-telephony-alias-members-add-modal.component.js]: Did you create a JIRA ticket ? 😄

**Comment by FredericEspiau** [comment]
How did this happen ?

File: client/components/telecom/telephony/alias/members/telecom-telephony-alias-members-add-modal.component.js (hunk 1 above)
Comment (JeremyDec) [reply client/components/telecom/telephony/alias/members/telecom-telephony-alias-members-add-modal.component.js]: For the fix, the one linked in the description

**Comment by JeremyDec** [comment]
@FredericEspiau this was deleted by error, and maybe too much confidence in the fact there will be no side effects.
@marie-j, hmm the fact is not the whole commit is revert. Some files can as been moved, some other has been deleted, and all this in the same commit.

**Comment by FredericEspiau** [comment]
@JeremyDec then you have two commits: `revert` then `refactor` :)

**Comment by JeremyDec** [comment]
@FredericEspiau problem, there is no refactor in this PR. I can set a revert message if you want, but that is not completely relevant.

For the second point, broken feature is one that has not been changed by the VOIP alias feature, while it depends on files that have been removed, for the purpose of VOIP Alias.
It hasn't been detected by test because the scope of VOIP alias feature do not integrate the broken part.

To be simple, some "non-regression test" could have protect us from this mistake.
Hope I was clear ;)

**Pull request merged**